roles/openshift_version/tasks/first_master.yml

```diff
@@ -13,20 +13,30 @@
 
 - include_tasks: "{{ l_first_master_version_task_file }}"
 
+# When double upgrade is in process, we want to set everything to match
+# openshift_verison.
 - block:
   - debug:
       msg: "openshift_pkg_version was not defined. Falling back to -{{ openshift_version }}"
   - set_fact:
-      openshift_pkg_version: -{{ openshift_version }}
+      openshift_pkg_version: "-{{ openshift_version }}"
   when:
-  - openshift_pkg_version is not defined or openshift_pkg_version == ""
-  - openshift_upgrade_target is not defined
+  - openshift_pkg_version is not defined or l_double_upgrade_cp
```

Comment:
During a double upgrade, this conditional is overriding the user provided `openshift_pkg_version`.

```yaml
# If I ask for
openshift_pkg_version: -3.9.0-0.42.0

# I get
openshift_pkg_version: -3.9.0
```